### PR TITLE
vertiv pdu mg02e4w1

### PR DESCRIPTION
**Pull request by Galileo77**
Please give a short description what your pull request is for

Detection of Vertiv Geist PDU type MG02E4W1

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/vertiv-pdu.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ icon: vertiv
 group: vertiv
 discovery:
     - sysObjectID:
+        - .1.3.6.1.4.1.21239.42.1.52
```
Comment (murrant) [inline includes/definitions/vertiv-pdu.yaml:8]: Just these two Vertiv PDU devices?  or should the sysObjectID be less specific?

Comment (Galileo77) [reply includes/definitions/vertiv-pdu.yaml:8]: We have only these two types of Vertiv Geist PDUs. With the sysObjectID - .1.3.6.1.4.1.21239.42.1.53 LibreNMS recognizes only one type. The second type needs the sysObjectID -.1.3.6.1.4.1.21239.42.1.52

Comment (murrant) [reply includes/definitions/vertiv-pdu.yaml:8]: Yes, but `.1.3.6.1.4.1.21239.42.1.` would detect both of those.  Adding devices individually is a losing game.

Comment (Galileo77) [reply includes/definitions/vertiv-pdu.yaml:8]: Ok i will try it on our test system this night.

Comment (Galileo77) [reply includes/definitions/vertiv-pdu.yaml:8]: Both devices are recognized, no problems so far.

**Review by murrant — APPROVED** [review]

**Pull request merged**